tests/adapters/structures/utils.py

```diff
@@ -7,9 +7,7 @@ def get_min_ver(dependency: str) -> str:
     setup_py = Path(__file__).parent.joinpath("../../../setup.py")
     with open(setup_py, "r") as setup_file:
         for line in setup_file.readlines():
-            min_ver = re.findall(
-                fr'"{dependency}((=|!|<|>|~)=|>|<)([0-9]+(\.[0-9]+){{,2}})"', line
-            )
+            min_ver = re.findall(fr'"{dependency}((=|!|<|>|~)=|>|<)(+.))"', line)
```

Comment:
A woopsie has snuck in 😅 
```suggestion
            min_ver = re.findall(fr'"{dependency}((=|!|<|>|~)=|>|<)(.+))"', line)
```

Reply:
Huh, tested this locally with `+.` and it seemed to work fine... I guess you'll get that suggestion in after all :P

Reply:
> Huh, tested this locally with `+.` and it seemed to work fine... I guess you'll get that suggestion in after all :P

Don't exactly know what `+.` means though 😅 `.` should be a wildcard for any character and `+` signifies that the immediately previous character should be repeated 1 to infinite times, i.e., there should be at least 1 character of any kind. So the reverse would be.. hmm... well, I'm still unsure what the `+` would be 😅

Reply:
There's a decent chance I typed it by hand into my interpreter when testing and then copied your version into the code...